Week-1/Exercise-1.2/css/styles.css

```diff
@@ -126,12 +126,13 @@ html,body
 }
 
 .footer-text{
-    position: absolute;
+    display: inline-block;    /*  changes made */
```

Comment:
@Globetrottercode you don't need to tag the changes, GitHub is exactly doing the Jon 🙂